pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -33,23 +33,10 @@ func addDefaultingFuncs(scheme *kruntime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManagerConfiguration) {
-	zero := metav1.Duration{}
-	if obj.ServiceController.ConcurrentServiceSyncs == 0 {
-		obj.ServiceController.ConcurrentServiceSyncs = 1
-	}
-	if obj.NodeStatusUpdateFrequency == zero {
-		obj.NodeStatusUpdateFrequency = metav1.Duration{Duration: 5 * time.Minute}
-	}
-}
-
 func SetDefaults_KubeControllerManagerConfiguration(obj *KubeControllerManagerConfiguration) {
 	zero := metav1.Duration{}
-	if len(obj.Controllers) == 0 {
-		obj.Controllers = []string{"*"}
-	}
-	if obj.EndPointController.ConcurrentEndpointSyncs == 0 {
-		obj.EndPointController.ConcurrentEndpointSyncs = 5
+	if obj.EndpointController.ConcurrentEndpointSyncs == 0 {
```

Comment:
Why removing settings for `Controllers`?

I'm afraid this will break existing clusters.

Reply:
Controllers was just moved to an other struct if you look closer, to the generic config